src/subscript/ecldiff2roff/ecldiff2roff.py

```diff
@@ -191,8 +191,11 @@ def ecldiff2roff_main(
             str(date_pair[0]),
             str(date_pair[1]),
         )
-        # Inplace substraction, prop1 = prop1 - prop2
-        prop1.values -= prop2.values
+
+        diffprop = xtgeo.GridProperty(
+            ncol=ecl_grid.ncol, nrow=ecl_grid.nrow, nlay=ecl_grid.nlay
```

Comment:
OK, FYI an altaernative and perhaps easier approach would:
`diffprop = prop1.copy()  # makes a new instance`